### PR TITLE
Kubernetes design document in the readme

### PR DESCRIPTION
**Pull request by travisn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,30 +181,51 @@ See [Building](https://github.com/rook/rook/wiki/Building) in the wiki for more
...
 
-![Overview](Documentation/media/cluster.png)
+![Rook Architecture on Kubernetes](Documentation/media/kubernetes.png)
 
-The Rook daemon `rookd` is a single binary that is self-contained and has all that is needed to bootstrap, scale
-and manage a storage cluster. `rookd` is typically compiled into a single static binary (just like most golang
-binaries) or a dynamic binary that takes a dependency on mostly libc. It can run in minimal containers, alongside a
-hypervisor, or directly on the host on most Linux distributions.
+The Rook operator is a simple container containing the `rook-operator` binary that has all that is needed to bootstrap
+and and monitor the storage cluster. The operator will start and monitor ceph monitor pods and a daemonset for the OSDs to
```
Comment (kokhang) [inline README.md]: duplicate word 'and'

File: README.md [hunk 2]
```diff
@@ -181,30 +181,51 @@ See [Building](https://github.com/rook/rook/wiki/Building) in the wiki for more
 
 ## Design
 
-A rook cluster is made up of one or more nodes each running the Rook daemon `rookd`. Containers and Pods can
-mount block devices and filesystems exposed by the cluster, or can use S3/Swift API for object storage. There is
-also a REST API exposed by `rookd` as well as a command line tool called `rook`.
+Rook is fully integrated with Kubernetes. Once Rook is running in the Kubernetes cluster, Kubernetes applications can
```
Comment (DanKerns) [inline README.md]: You might strike the first sentence.  I'm not sure I know what "fully integrated" means.

File: README.md [hunk 3]
```diff
@@ -181,30 +181,51 @@ See [Building](https://github.com/rook/rook/wiki/Building) in the wiki for more
...
+![Rook Architecture on Kubernetes](Documentation/media/kubernetes.png)
 
-The Rook daemon `rookd` is a single binary that is self-contained and has all that is needed to bootstrap, scale
-and manage a storage cluster. `rookd` is typically compiled into a single static binary (just like most golang
-binaries) or a dynamic binary that takes a dependency on mostly libc. It can run in minimal containers, alongside a
-hypervisor, or directly on the host on most Linux distributions.
+The Rook operator is a simple container containing the `rook-operator` binary that has all that is needed to bootstrap
+and and monitor the storage cluster. The operator will start and monitor ceph monitor pods and a daemonset for the OSDs to
+which provides basic RADOS storage, as well as a deployment for a RESTful API service. When requested through the api service,
+object storage (S3/Swift) is enabled by starting a deployment for RGW, while a file server is enabled with a deployment for MDS.
```
Comment (DanKerns) [inline README.md]: ...deployment for RGW, while a **shared file system** is enabled with a deployment for MDS.

File: README.md [hunk 4]
```diff
@@ -181,30 +181,51 @@ See [Building](https://github.com/rook/rook/wiki/Building) in the wiki for more
...
 
+The operator will monitor the storage daemons to ensure the cluster is healthy, to start more ceph mons when necessary, to 
+failover mons, make adjustments as the cluster grows or shrinks, etc. The operator will also watch for desired state changes 
+requested by the api service and apply the changes.
+
+The Rook daemons (Mons, OSDs, RGW, and MDS) are compiled to a single binary `rookd`, and included in a minimal container.
 `rookd` uses an embedded version of Ceph for storing all data -- there are no changes to the data path. An embedded version
```
Comment (DanKerns) [inline README.md]: "embedded version of Ceph" -> Embedded Ceph.  I'd strike the next sentence "An embedded version of ceph..."

File: README.md [hunk 5]
```diff
@@ -181,30 +181,51 @@ See [Building](https://github.com/rook/rook/wiki/Building) in the wiki for more
...
+See [Design](https://github.com/rook/rook/wiki/Design) wiki for more details.
+
+### Standalone Rook
+
+Rook also supports an environment where there is no orchestration platform. Rook comes with its own basic orchestration 
+engine based on Etcd that is activated when not running in Kubernetes. 
+
+![Standalone Rook Architecture](Documentation/media/standalone.png)
+
+The Rook daemon `rookd` is a single binary that is self-contained and has all that is needed to bootstrap, scale
```
Comment (DanKerns) [inline README.md]: The Rook daemon, `rookd`, is a... (add comma's)

File: README.md (hunk 1 above)
Comment (travisn) [inline README.md]: good catch

File: README.md (hunk 3 above)
Comment (travisn) [inline README.md]: will update

File: README.md (hunk 4 above)
Comment (travisn) [inline README.md]: yes, i'll remove the redundancy

File: README.md (hunk 5 above)
Comment (travisn) [inline README.md]: will do

File: README.md (hunk 2 above)
Comment (travisn) [inline README.md]: i'll make this more descriptive.

**Comment by travisn** [comment]
this content will come with a new PR today

**Pull request closed without merging**